crates/driver/src/infra/solver/dto/notification.rs

```diff
@@ -89,11 +89,28 @@ impl Notification {
 #[serde(rename_all = "camelCase")]
 pub struct Notification {
     auction_id: Option<String>,
-    solution_id: Option<u64>,
+    solution_id: Option<SolutionId>,
     #[serde(flatten)]
     kind: Kind,
 }
 
+#[serde_as]
+#[derive(Debug, Serialize)]
+#[serde(untagged)]
+pub enum SolutionId {
+    Single(u64),
+    Merged(Vec<u64>),
+}
```

Comment:
Why is this added? I don't see that we changed the function `notify_id()` function which is used for sending notification.